conf/config.go

```diff
@@ -69,11 +69,34 @@ const (
 // if configuration file found and loaded, state doesn't changed
 var state configStatus = configGood
 
-/*
-   This is the private helper function that sets up viper. This function is
-   called by the init() function only once during initialization of the package.
-*/
-func setup(locations ...string) (config, configStatus) {
+// init: First thing to run when this package is loaded by the binary.
+// Even if multiple packages import conf, this will be called and ran ONLY once.
+func init() {
```

Comment:
plugged init() into the top of the file and added comments. eventually, it would be beneficial to make this a part of application startup instead initializing immediately.